src/external-repos.md

```diff
@@ -80,6 +80,16 @@ switch to its repository checkout directory in your terminal).
 
 #### Performing push
 
+> NOTE:
+> If you use Git protocol to push to your fork of `rust-lang/rust`,
+> ensure that you have this entry in your Git config,
+> else the 2 steps that follow would prompt for a username and password:
+>
+> ```
+> [url "git@github.com:"]
```

Comment:
This is documented here: https://github.com/rust-lang/josh-sync?tab=readme-ov-file#git-peculiarities I wanted to centralize the documentation in this repository, and link to it from rdg and other sources.